.github/workflows/pr-title-check.yml

```diff
@@ -0,0 +1,62 @@
+name: Check PR Title and Issue Consistency
+
+on:
+  pull_request:
+    types: [opened, edited, reopened, synchronize]
```

Comment:
This tell github when to run this job. Here, it runs when pull requests (PR) are opened, reopened, edited (description and title) or synchronized (pushed to)

Reply:
Feel free to resolve if you understand